composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "boedah/robo-drush",
+    "name": "greg-1-anderson/robo-drush",
```

Comment:
Yeah, that is temporary so that I can pull this branch in to my projects before this is merged. There are other solutions, but they require me to be less lazy. :) Just need to remember to fix it before we're done here!